src/mlpack/methods/ann/layer/radial_basis_function_impl.hpp

```diff
@@ -12,33 +12,38 @@
 #define MLPACK_METHODS_ANN_LAYER_RBF_IMPL_HPP
 
 // In case it hasn't yet been included.
-#include "dropout.hpp"
+#include "radial_basis_function.hpp"
 
 namespace mlpack {
 namespace ann /** Artificial Neural Network. */ {
 
-template<typename InputDataType, typename OutputDataType>
-RBF<InputDataType, OutputDataType>::RBF() :
+template<typename InputDataType, typename OutputDataType,
+         typename Activation>
+RBF<InputDataType, OutputDataType, Activation>::RBF() :
     inSize(0),
     outSize(0)
 {
   // Nothing to do here.
 }
 
-template<typename InputDataType, typename OutputDataType>
-RBF<InputDataType, OutputDataType>::RBF(
+template<typename InputDataType, typename OutputDataType,
+         typename Activation>
+RBF<InputDataType, OutputDataType, Activation>::RBF(
     const size_t inSize,
     const size_t outSize,
-    arma::mat& centres) :
+    arma::mat& centres,
+    Activation activation) :
     inSize(inSize),
     outSize(outSize),
-    centres(centres)
+    centres(centres),
+    activation(activation)
```

Comment:
Can you implement it how it is implemented in the BaseLayer ? I don't think we need to pass it through the constructor.